apps/ui/src/views/Proposal/Discussion.vue

```diff
@@ -46,15 +46,15 @@ onMounted(async () => {
         class="py-4 border-b last:border-b-0"
       >
         <div class="max-w-[680px] mx-auto">
-          <div class="flex">
+          <div class="flex gap-2.5 items-center">
             <a
               :href="reply.user_url"
               target="_blank"
-              class="w-[32px] h-[32px] shrink-0 mr-2.5 mt-0.5"
+              class="shrink-0 rounded-full"
```

Comment:
Any reason for `roudend-full` on `a` tag? image is already rounded so only image will be clickable, I tried removing and it is same functionality without 
```suggestion
              class="shrink-0"
```

Reply:
For the outline when using TAB key

Reply:
ahh